### DISCUSSION TITLE
FNF mobile web version

### DISCUSSION THREAD

**Discussion by luckydog7** [luckydog7/Funkin-android#1399] [General]
https://luckydog7.github.io/funkinmobile/

**Comment by Ruv023** [comment]
This is amazing. My phone doesnt crash at all no matter what song I play. Love it

**Reply by Pixelbeep** [reply]
No black box it's perfect

**Comment by DiosDonutSlave** [comment]
I think some good mods to put in would be ofc, whitty, maybe tricky depending on the capability of the textures in browser, and zardy, to start

**Comment by FrazixYTP** [comment]
Poggers

**Comment by Hurbmoment** [comment]
i wanna make this mod added: SARVENTE MOD.

**Comment by blazingcody** [comment]
It went to aw snap in the browser version of this

**Comment by claus296** [comment]
Ayyyy it works on iOS too

**Comment by ghost** [comment]
Will this get updates?

**Reply by FrazixYTP** [reply]
Yes

**Comment by Hurbmoment** [comment]
this shit dead as hell XD

**Comment by Xavierra12345** [comment]
Are you gonna make downloads for the html zip file so we can make mods

**Reply by Xavierra12345** [reply]
Oh wait its this

**Reply by Xavierra12345** [reply]
https://github.com/luckydog7/luckydog7.github.io
